packages/Babylonian-Core.package/BPExampleTraceInformation.class/instance/trace..st

```diff
@@ -1,4 +1,6 @@
 accessing
 trace: anObject
-
-	^ trace := anObject
\ No newline at end of file
+	
+	trace := anObject.
+	self changed: #newTrace with:anObject.
```

Comment:
Maybe we can move most of these on level up to BPExample and then we can simply observe the example.